dpnp/dpnp_iface_bitwise.py

```diff
@@ -161,13 +167,14 @@ def bitwise_and(x1, x2, dtype=None, out=None, where=True, **kwargs):
     [2, 4, 16]
```

Comment:
Could you please simplify and update the example a bit, like (and add two more examples):
```
    >>> np.bitwise_and(x1, x2)
    array([ 2,  4, 16])

    >>> a = np.array([True, True])
    >>> b = np.array([False, True])
    >>> np.bitwise_and(a, b)
    array([False,  True])

    The ``&`` operator can be used as a shorthand for ``bitwise_and`` on
    :class:`dpnp.ndarray`.

    >>> x1 & x2
    array([ 2,  4, 16])
```